coordinator/src/main/java/com/tencent/rss/coordinator/AccessManager.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.coordinator;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.tencent.rss.common.util.RssUtils;
+
+public class AccessManager {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AccessManager.class);
+
+  private final CoordinatorConf coordinatorConf;
+  private final ClusterManager clusterManager;
+  private List<AccessChecker> accessCheckers = Lists.newArrayList();
+
+  public AccessManager(CoordinatorConf conf, ClusterManager clusterManager) throws RuntimeException {
+    this.coordinatorConf = conf;
+    this.clusterManager = clusterManager;
+    init();
+  }
+
+  private void init() throws RuntimeException {
```

Comment:
Suggest implement init method as following style:
1. get value of checkers
2. loop every checker and skip empty value
3. if there has no checker, throw exception

Reply:
This function is actually this style, get and parse the checkers conf, trim and split empty chars, load and throw an exceptions if fail to load or empty checker